hawkular-inventory-paths/pom.xml

```diff
@@ -28,6 +28,13 @@
 
   <dependencies>
     <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-annotations</artifactId>
+      <version>${version.io.swagger}</version>
```

Comment:
What is the relationship of `io.swagger:swagger-annotations` to `com.wordnik:swagger-annotations` that we manage in parent? https://github.com/hawkular/hawkular-parent-pom/blob/41/pom.xml#L322-L323

Are `io.swagger:swagger-annotations` newer and better? Should current occurrences of `com.wordnik:swagger-annotations` be replaced by `io.swagger:swagger-annotations` ?